lib/rubocop/cop/minitest/multiple_assertions.rb

```diff
@@ -24,6 +24,14 @@ module Minitest
       #     def test_another_asserts_once
       #       assert_empty(array)
       #     end
+      #
+      #     def test_conditional_assertion
+      #       if condition
+      #         assert foo
+      #       else
+      #         assert bar
+      #       end
+      #     end
```

Comment:
I worry about losing the symmetry between `#bad` and `#good` examples. Can you replace the added example code with a description of the cop using natural language?